data/maps/Route117/scripts.inc

```diff
@@ -1,12 +1,60 @@
 .set LOCALID_DAYCARE_MAN, 3
+.set LOCALID_DAYCARE_MON_0, 25
+.set LOCALID_DAYCARE_MON_1, 26
 
 Route117_MapScripts::
 	map_script MAP_SCRIPT_ON_TRANSITION, Route117_OnTransition
 	.byte 0
 
+.macro getdaycaregfx var0:req var1:req var2:req var3:req
+callfunc ScrFunc_getdaycaregfx
+.2byte \var0
+.2byte \var1
+.2byte \var2
+.2byte \var3
+.endm
```

Comment:
Not that I know of, tbh

Reply:
Then these should be moved

Reply:
It's only going to be used on this one map, though...

Reply:
Still, if someone wants to use these on their own Daycare map, having them in the general file would make sense instead of trying to recover these lines from a potentially deleted file.